sim.py

```diff
@@ -59,9 +113,9 @@ def run_sims(args, iterations, talent, covenant):
             raidbots(api_key, profile_location,
                      config["simcBuild"], output_location, profile_name_with_dir, iterations)
         elif weight == 0:
-            print("{0} has a weight of 0. Skipping file.".format(output_name))
+            print("-- {0} has a weight of 0. Skipping file.".format(output_name))
```

Comment:
what do these dashes do?

Reply:
This dashes don't do much :), just formatting the output a bit: So it looks like this
Simming 6 out of 18
-- consumables_lm_na_2.json has a weight of 0. Skipping file.

Did this do have a "better" overview in the shell when there are around 100 - 200 lines to see what happens, and where i have problems. I produce similar lines for example when something in the sim itself run's to an error

Simming 1 out of 18
-- consumables_hm_na_1.json has an error. Skipping file.
-- simc: /home/espo/projects/simc/engine/action/sc_action.cpp:93: virtual void {anonymous}::queued_action_execute_event_t::execute(): Assertion `action->cooldown->ready <= sim().current_time()' failed.

So i can see at the end of an sim when i return to the pc quite quickly what happend and what got skipped and so on.